src/WebJobs.Script/Description/Node/NodeFunctionInvoker.cs

```diff
@@ -184,7 +183,7 @@ static NodeFunctionInvoker()
                 inputs.Add(value);
             }
 
-            executionContext["inputs"] = inputs;
+            executionContext["_inputs"] = inputs;
```

Comment:
Renaming some of these temp system props with _ to avoid conflicts with user properties